stories/Input/default.js

```diff
@@ -241,7 +241,7 @@ storiesOf('Inputs', module)
           base="dark"
           mask="111-111-111"
           placeholder="Type your phone number"
-          type="text"
+          type="phone"
```

Comment:
hm...

Reply:
it shouldn't be tel?
```suggestion
          type="tel"
```